script.lazytv/resources/skins/Default/media/.directory

```diff
@@ -0,0 +1,10 @@
+[Dolphin]
```

Comment:
what is this file for? If not used by the addon please remove from the submission